### PR TITLE
FEAT: Added Responsive pages & Basic Routing

### PR DESCRIPTION
**Pull request by kotesh-arya**
## Detailed features of this PR:

- Responsive pages.
- MODULAR STYLING for limiting the styling scope of each individual component.
- Basic ROUTING for navigation.

### CODE REVIEW

**Review by MaruthiPothuganti — APPROVED** [review]
Good UI
Modular Code and 
Clean code. PR looks good to merge.

File: src/pages/Singlevideo/Singlevideo.module.css [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    color: var(--white);
+    margin-left: 0;
+    width: 100%;
+    min-height: 100vh;
+  }
+  .video-section {
+    margin: 0;
+    width: 110%;
+    /* background-color: yellow ; */
+  }
```
Comment (swethaogeti) [inline src/pages/Singlevideo/Singlevideo.module.css:232]: remove comments !

File: src/pages/Singlevideo/Singlevideo.module.css [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+    background-color: var(--page-bg-color);
+    /* background-color: red; */
+    color: var(--white);
+    margin-left: 0;
+    width: 100%;
+    min-height: 100vh;
+  }
+  .video-section {
+    margin: 0;
+    width: 115%;
```
Comment (swethaogeti) [inline src/pages/Singlevideo/Singlevideo.module.css]: width:115% check this ! more than 100%

File: src/pages/Videolisting/Navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import logo from "../Navbar/assets/streamStar-logo.png";
+import { Link } from "react-router-dom";
+import { FaMicrophone, FaSearch } from "react-icons/fa";
+import { GiHamburgerMenu } from "react-icons/gi";
+import NavbarCSS from "../Navbar/Navbar.module.css";
+console.log(`${NavbarCSS["nav-center"]}${NavbarCSS["flex-div"]}`);
```
Comment (swethaogeti) [inline src/pages/Videolisting/Navbar/Navbar.js]: remove console.log

**Review by swethaogeti — APPROVED** [review]
code looks clean ! remove commented code , using modular css is awsome ! good to merge

File: src/pages/Singlevideo/Singlevideo.module.css (hunk 2 above)
Comment (kotesh-arya) [reply src/pages/Singlevideo/Singlevideo.module.css]: Adjusted it, Thanks for pointing it 👍🏻

File: src/pages/Videolisting/Navbar/Navbar.js (hunk 3 above)
Comment (kotesh-arya) [reply src/pages/Videolisting/Navbar/Navbar.js]: Removed :)

**Pull request merged**